Ginger/GingerCoreNET/CentralExecutionLogger/CentralExecutionLoggerHelper.cs

```diff
@@ -0,0 +1,150 @@
+                cfg.CreateMap<LiteDbRunner, AccountReportRunner>();
+                
+                cfg.CreateMap<LiteDbBusinessFlow, AccountReportBusinessFlow>();
+                cfg.CreateMap<LiteDbActivityGroup, AccountReportActivityGroup>();
+                cfg.CreateMap<LiteDbActivity, AccountReportActivity>();
+                cfg.CreateMap<LiteDbAction, AccountReportAction>();
+                //cfg.CreateMap<KeyValuePair<string, int>, AccountReport.Contracts.Helpers.DictObject>().ConvertUsing(x =>
+                cfg.CreateMap<KeyValuePair<string, int>, AccountReport.Contracts.Helpers.DictObject>()
+                .ForMember(dest => dest.Key, src => src.MapFrom(x => x.Key))
+                .ForMember(dest => dest.Value, src => src.MapFrom(x => x.Value));
+
+            });
+
+            IMapper iMapper = config.CreateMapper();
+
+            var destination = iMapper.Map<LiteDbRunSet, AccountReportRunSet>(runSet);
+            
+            //destination.ElapsedEndTimeStamp = runSet.Elapsed;
+            return destination;
+        }
+
+        public async Task SendRunsetExecutionDataToCentralDBAsync(AccountReportRunSet accountReportRunSet)
+        {
+            Reporter.ToStatus(eStatusMsgKey.PublishingToCentralDB, null,  "Publishing Execution data to central DB");
+            RestRequest restRequest = (RestRequest)new RestRequest("/AccountReport/AddRunSetToAccountDB/", Method.POST) { RequestFormat = RestSharp.DataFormat.Json }.AddJsonBody(accountReportRunSet);
+            string message = string.Format(" execution data to Central DB for Runset: {0} , Execution Id:{1}", accountReportRunSet.Name, accountReportRunSet.ExecutionId);
+            try
+            {            
+                IRestResponse response = await restClient.ExecuteAsync(restRequest);
+                if(response.IsSuccessful)
+                {                    
+                    Reporter.ToLog(eLogLevel.INFO, "Successfully sent "+message);
+                }
+                else
+                {
+                    Reporter.ToLog(eLogLevel.WARN, "Failed to send "+message +"Response: "+ response.Content);
+                }                
+            }
+            catch(Exception ex)
+            {
+                Reporter.ToLog(eLogLevel.ERROR, "Exception when sending "+message, ex);
+            }
+            finally
+            {
+                Reporter.HideStatusMessage();
+            }
+        }
+
+        public async Task SendScreenShotsToCentralDBAsync(Guid executionId, List<string> filePaths)
+        {
+            long fileSize = 0;
+    
+            List<string> temp = new List<string>();
+
+            foreach (string screenshot in filePaths)
+            {
+                fileSize += new System.IO.FileInfo(screenshot).Length;
+                temp.Add(screenshot);
+                if ((5 * 1000000) < fileSize) // 5 MB
+                {
+                    await UploadImageAsync(executionId,temp);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5632404)